apps/bridge-dapp/src/pages/PageBridge.tsx

```diff
@@ -147,169 +166,190 @@ const PageBridge = () => {
+            }
+            // The customMainComponent alters the global mainComponent for display.
+            // Therfore, if the customMainComponent exists (input selected) then hide the base component.
+            className={cx(
+              'max-w-[550px] bg-mono-0 dark:bg-mono-180 p-4 rounded-lg space-y-4 grow',
+              customMainComponent ? 'hidden' : 'block'
+            )}
+          >
+            <TabsList aria-label="bridge action" className="mb-4">
+              <TabTrigger value="Deposit">Deposit</TabTrigger>
+              <TabTrigger value="Transfer">Transfer</TabTrigger>
+              <TabTrigger value="Withdraw">Withdraw</TabTrigger>
+            </TabsList>
+            <TabContent value="Deposit">
+              <DepositContainer {...sharedBridgeTabContainerProps} />
+            </TabContent>
+            <TabContent value="Transfer">
+              <TransferContainer {...sharedBridgeTabContainerProps} />
+            </TabContent>
+            <TabContent value="Withdraw">
+              <WithdrawContainer {...sharedBridgeTabContainerProps} />
+            </TabContent>
+          </TabsRoot>
+
+          <div>
+            {/** Transaction Queue Card */}
+            {isDisplayTxQueueCard && (
+              <TransactionQueueCard
+                className="w-full mb-4 max-w-none"
+                transactions={txPayloads}
+              />
+            )}
+
+            {/** Education cards */}
+            <div className="p-9 max-w-[386px] bg-blue-10 dark:bg-blue-120 rounded-lg">
+              <Typography
+                variant="body1"
+                fw="semibold"
+                className="text-blue-70 dark:text-blue-50"
               >
-                Support
-              </Button>
+                Learn about what makes Webb private and how this makes using it
+                different from other bridges.
+              </Typography>
+
+              <div className="grid grid-cols-2 gap-4 mt-4">
+                <Button
+                  leftIcon={<CoinIcon size="lg" className="!fill-current" />}
+                  href="https://docs.webb.tools" // TODO: Determine link here
+                  target="_blank"
+                  variant="link"
+                >
+                  Usage Guide
+                </Button>
+
+                <Button
+                  leftIcon={<BlockIcon size="lg" className="!stroke-current" />}
+                  href="https://docs.webb.tools" // TODO: Determine link here
+                  target="_blank"
+                  variant="link"
+                >
+                  FAQ
+                </Button>
+
+                <Button
+                  leftIcon={
+                    <HelpLineIcon size="lg" className="!fill-current" />
+                  }
+                  href="https://docs.webb.tools"
+                  target="_blank"
+                  variant="link"
+                >
+                  Get Started
+                </Button>
```

Comment:
@AtelyPham I will create an issue for this component as we will need to include different links depending if the user is on the deposit or withdraw page.